.eslintrc

```diff
@@ -31,7 +31,6 @@
     "prefer-const": "error",
     "vars-on-top" : "error",
     "no-cond-assign": "error",
-    "no-console" : "warn",
```

Comment:
Since we now include console errors in our code, I removed this lint rule to remove noise.